.circleci/config.yml

```diff
@@ -17,6 +22,13 @@ commands:
           command: |
             cd scripts && /usr/bin/bash ./machine-launch-script.sh
 
+  install-fabric:
+    description: "Installs fabric for manager-managing CI container"
```

Comment:
Maybe call this `install-ci-python-reqs` instead?  For the description "Install minimal python dependencies for CI"?